src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2851,10 +2851,10 @@ private static bool IsAvailableFarEastCodePage(uint codePage)
         private static bool IsAnyDBCSCharSet(uint charSet)
         {
             const uint SHIFTJIS_CHARSET = 128;
-            const uint HANGEUL_CHARSET = 129;
+            const uint HANGUL_CHARSET = 129;
```

Comment:
FWIW, I generally try to change things to have a single consistent spelling. If there was a dominant spelling in the project, e.g. an en-GB spelling of `colour`, then I would have settled on that. Currently there are three spellings used about evenly. Given that, my default was to pick the single most commonly used spelling (in this case, a word with which I'm familiar, although I will also ask Google to confirm prevalence).

`Hangul`: https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=hangul
`Hangeul`: https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=HANGEUL_CHARSET&type=Code
`Hangule`: https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=hangule&type=Code